eZ/Bundle/EzPublishRestBundle/Tests/Functional/ContentTest.php

```diff
@@ -414,4 +443,104 @@ public function testCreateView()
+     *
+     * @return string
+     */
+    private function createMultiLanguageContent()
+    {
+        $this->ensureLanguageExists('eng-US', 'English (United States)');
+
+        $request = $this->createHttpRequest(
+            'POST',
+            '/api/ezp/v2/content/objects',
+            'ContentCreate+xml',
+            'ContentInfo+json'
+        );
+        $name = 'Multi Language Content';
+        $modificationDate = date('c');
+        $body = <<< XML
+<?xml version="1.0" encoding="UTF-8"?>
+<ContentCreate>
+  <ContentType href="/api/ezp/v2/content/types/1" />
+  <mainLanguageCode>eng-GB</mainLanguageCode>
+  <LocationCreate>
+    <ParentLocation href="/api/ezp/v2/content/locations/1/2" />
+    <priority>0</priority>
+    <hidden>false</hidden>
+    <sortField>PATH</sortField>
+    <sortOrder>ASC</sortOrder>
+  </LocationCreate>
+  <Section href="/api/ezp/v2/content/sections/1" />
+  <alwaysAvailable>true</alwaysAvailable>
+  <User href="/api/ezp/v2/user/users/14" />
+  <modificationDate>{$modificationDate}</modificationDate>
+  <fields>
+    <field>
+      <fieldDefinitionIdentifier>name</fieldDefinitionIdentifier>
+      <languageCode>eng-GB</languageCode>
+      <fieldValue>BrE {$name}</fieldValue>
+    </field>
+    <field>
+      <fieldDefinitionIdentifier>name</fieldDefinitionIdentifier>
+      <languageCode>eng-US</languageCode>
+      <fieldValue>AmE {$name}</fieldValue>
+    </field>
+  </fields>
+</ContentCreate>
+XML;
+        $request->setContent($body);
+
+        $response = $this->sendHttpRequest($request);
+
+        self::assertHttpResponseCodeEquals($response, 201);
+        self::assertHttpResponseHasHeader($response, 'Location');
+
+        $href = $response->getHeader('Location');
+        $this->addCreatedElement($href);
+
+        // publish created content
+        $response = $this->sendHttpRequest(
+            $this->createHttpRequest('PUBLISH', "{$href}/versions/1")
+        );
+        self::assertHttpResponseCodeEquals($response, 204);
+
+        return $href;
+    }
+
+    /**
+     * Get Content Object Versions XML as DOMDocument.
+     *
+     * @param string $contentObjectURI
+     *
+     * @return \DOMDocument
+     */
+    private function loadVersionJSON($contentObjectURI, $versionNo)
+    {
+        $request = $this->createHttpRequest(
+            'GET',
+            "{$contentObjectURI}/versions/{$versionNo}",
+            '',
+            'Version+json'
+        );
+        $response = $this->sendHttpRequest($request);
+        self::assertHttpResponseCodeEquals($response, 200);
+
+        return json_decode($response->getContent(), true);
+    }
+
+    /**
+     * Check if the given language exists and create it if it doesn't.
+     *
+     * @param $languageCode
+     * @param $name
+     */
+    private function ensureLanguageExists($languageCode, $name)
+    {
+        self::markTestIncomplete('@todo implement EZP-21171');
```

Comment:
Language manipulation via REST API is not available yet, so we're not able to test REST endpoint as initial clean installer data contains one language only.